### PR TITLE
Minor bug fixes to Celery, Partners Export and Trips

### PR DESCRIPTION
**Pull request by ntrncic**
Fixed celery periodic tasks
Fixed 500 error for Partners Export by Donor
Fixed Trip resend approval email if trip is set back to submitted.

### CODE REVIEW

File: EquiTrack/EquiTrack/settings/base.py [hunk 1]
```diff
@@ -129,7 +129,7 @@
 djcelery.setup_loader()
 BROKER_URL = os.environ.get('REDIS_URL', 'redis://localhost:6379/0')
 CELERY_RESULT_BACKEND = 'djcelery.backends.database:DatabaseBackend'
-CELERY_BEAT_SCHEDULER = 'djcelery.schedulers.DatabaseScheduler'
+CELERYBEAT_SCHEDULER = 'djcelery.schedulers.DatabaseScheduler'
```
Comment (jamescw) [inline EquiTrack/EquiTrack/settings/base.py]: oh man all that time...

File: EquiTrack/partners/exports.py [hunk 2]
```diff
@@ -86,25 +86,30 @@ def get_title(self):
...
-                        'Locality': loc.locality.name,
-                        'Cad Code': loc.locality.cad_code,
-                        'x': loc.location.point.x,
-                        'y': loc.location.point.y
-                    }
-                )
-
-        data = tablib.Dataset(headers=locs[0].keys())
+
+        if dataset._Dataset__headers:
```
Comment (jamescw) [inline EquiTrack/partners/exports.py]: any method with an underscore means in should be private and could change if we upgrade the library, anything else we can use here? Did you try just `if dataset:`?

Comment (ntrncic) [inline EquiTrack/partners/exports.py]: yes I tried a bunch of things always get error "not iterable".
This is the only one that worked for now. I'll look and see what else can be used instead.

Comment (jamescw) [inline EquiTrack/partners/exports.py]: see if there are methods on the dataset, I think there is a `.headers()` which will return a list

File: EquiTrack/EquiTrack/settings/base.py (hunk 1 above)
Comment (ntrncic) [inline EquiTrack/EquiTrack/settings/base.py]: it's good we're not sitting together, I would've thrown a ball at you at that moment.

**Pull request merged**